student.rb

```diff
@@ -0,0 +1,14 @@
+require_relative('./person')
+
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(classroom)
+    @classroom = classroom
+    super
+  end
```

Comment:
- [ ] Your `initialize` method here is missing the other parameters just as we have in the Person `initialize` method: `age`, `name` and `parent_permission`. Kindly add them.

- [ ] Pass on these other parameters to the Person class though the `super()` method, like so: `super(age. name, parent_permission)`